INSTALL.md

```diff
@@ -368,7 +368,16 @@ cd filter_audio
 make -j$(nproc)
 sudo make install
 ```
-
+### Sqlcipher compiling, For Fedora users and possibly others.
+```bash
+git clone https://github.com/sqlcipher/sqlcipher
+cd sqlcipher
+autoreconf -if
+./configure
+make -j$(nproc)
+sudo make install
+cd ..
+````
```

Comment:
@antis81 um, but it seems that it might? I.e. toxcore's dependency (`libtool`[1](http://rpmfind.net/linux/RPM/fedora/devel/rawhide/x86_64/l/libtool-2.4.6-7.fc24.x86_64.html)) pulls in `autoconf`[2](http://rpmfind.net/linux/RPM/fedora/23/i386/a/autoconf-2.69-21.fc23.noarch.html), which provides `autoreconf` required by `sqlcipher` to compile..

@linux-modder could you change the part `compiling, For Fedora users and possibly others.` to `(Fedora)`?

Also, please add interlinking, in similar way to how it's done for `### FFmpeg` section.